src/main/java/HeadersRestTemplateService.java

```diff
@@ -0,0 +1,28 @@
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpHeaders;
+import org.springframework.stereotype.Service;
+
+@Service
+public class HeadersRestTemplateService {
+
+    @Value("${github.token}")
+    private String githubToken;
+
+    @Value("${openai.api.key}")
+    private String openaiApiKey;
+
+    public HttpHeaders createGitHubHeaders() {
+        HttpHeaders headers = new HttpHeaders();
+        headers.set("Authorization", "Bearer " + githubToken);
+        headers.set("Content-Type", "application/json");
```

Comment:
The Content-Type header is set to application/json in both methods, which is redundant. This could be extracted to a common method or constant. (Severity: Low)